boot.php

```diff
@@ -9,10 +9,6 @@
  * file that was distributed with this source code.
  */
 
-if (!ini_get("auto_detect_line_endings")) {
-    ini_set("auto_detect_line_endings", '1');
-}
```

Comment:
Ich würde das ganz entfernen. Eingefügt hattest du es hier: https://github.com/tbaddade/redaxo_sprog/commit/9c3c64573dcc789b578f8a3ff5efe9e803e008e3
Benötigt wird es aber nur für Dateien, die die alten Mac-Line-Endings (`\r`) haben. Also Dateien, die noch vor OS X erstellt wurden, in Mac OS 9 oder früher. Seit OS X nutzt Mac ja die Unix-Line-Endings `\n`.

https://php.watch/versions/8.1/auto_detect_line_endings-ini-deprecated

Wenn es dir lieber ist, könnten wir aber auch eine dieser Varianten nutzen:
- Mit PHP-Weiche, ini-Option nur bei PHP<8.1 setzen
- Mit PHP-Weiche, ini-Option nur bei PHP<9 setzen (da wird die Option ja erst ganz entfernt) und `@` nutzen um die Deprecations seit PHP 8.1 zu unterdrücken